packages/@uppy/dashboard/src/components/AddFiles.tsx

```diff
@@ -305,18 +328,22 @@ class AddFiles extends Component {
   }
 
   private renderSourcesList = (
-    acquirers: $TSFixMe,
-    disableLocalFiles: $TSFixMe,
+    acquirers: TargetWithRender[],
+    disableLocalFiles: boolean,
   ) => {
     const { showNativePhotoCameraButton, showNativeVideoCameraButton } =
       this.props
 
-    let list = []
+    type RenderListItem = { key: string; elements: ComponentChild }
+    let list: RenderListItem[] = []
 
     const myDeviceKey = 'myDevice'
 
     if (!disableLocalFiles)
-      list.push({ key: myDeviceKey, elements: this.renderMyDeviceAcquirer() })
+      list.push({
+        key: myDeviceKey,
+        elements: this.renderMyDeviceAcquirer(),
+      })
```

Comment:
I dont't understand why prettier changed this 🤔 different configuration maybe?

Reply:
It's me!
I think it's easier to read when it's more consistent & symmetrical (see similar lines below). I throw my eye over these 20 lines without actually reading them, and know that the same thing happens in all of them.

Reply:
Alright, ideally we would land this separately so it doesn't pollute git blame and simplify the reviewing – but no big deal if we keep it here.